css/index.css

```diff
@@ -1,3 +1,461 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+a {
+	text-decoration: none;
+	color: gray;
+	font-size: 14px;
+}
+
+h1 {
+    font-size: 72px;
+    text-align: center;
+    margin-bottom: 2px;
+}
+
+button {
+    margin-top: 23px;
+    padding: 6px 41px;
+    font-size: 16px;
+    background-color: white;
+    border: 1px black solid;
+}
+
+.container {
+    width: 100%;
+    margin: 0 auto;
+    max-width: 1024px;
+}
+
+/* -------------- HEADER starts here -------------- */
+header {
+    width: 100%;
+    margin-top: 0;
+}
+
+        /* ---- HEADER-TOP starts here ----- */
+header > div.header-top {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-top: 27px;
+}
+                /* --- NAVIGATION starts here --- */
+nav {
+    display: flex;
```

Comment:
great use of display flex but I would like to see you use short hand for your margin here. This could have been written as margin: 4px 26px 0 42px. It helps to get a better overview of your code when it has less lines. Soon you'll be writing 100s of lines of javascript and you'll be happy when you can use a form of short hand to limit how much is on your screen at one time.